dictionaries/en_shared/dict/en-shared.txt

```diff
@@ -31,6 +31,8 @@ hydrophobe
 hyponym
 hyponyms
 iPads
+pregenerate
```

Comment:
Please add these words to [src/en-shared.txt](https://github.com/streetsidesoftware/cspell-dicts/tree/main/dictionaries/en_shared/src/en-shared.txt). They will get overwritten here.

This directory is auto generated. I should just remove it.

Reply:
My mistake, the right location is https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt